src/App.js

```diff
@@ -1,23 +1,40 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import Character from "./components/Character/Character";
+import Display from "./components/Display/Display";
+import Picker from "./components/Picker/Picker";
+import React, { useState } from "react";
 
 function App() {
+  const [head, setHead] = useState("");
```

Comment:
Your console has errors on the deployed site because it's trying to load images but the state for head, middle, and pants are set to empty strings. You can give these baseline images to use to avoid these errors.